3-0-libraries.md

```diff
@@ -248,14 +248,73 @@ summed when displayed on this page.
 
 # Troubleshooting
 
-1. How do I correct an index on a library?
-1. What if I assign a library to the wrong parent aliquot?
-1. What if I forget to put a library dilution in a pool?
-1. How do I change the targeted sequencing type on a library?
-1. How can I add a new targeted sequencing type, kit, or anything else in drop-down menus?
-1. How do I make bulk orders?
-1. How are matrix tube barcodes assigned to tubes?
-1. What is the importance of selecting a study (?) since it prevents the Sequencing stage from adding pools to the lane containers?
+## How do I correct an index on a library?
+The index can be changed on either the individual library page or the bulk edit page.
+
+1. From _Tracking_, _Library_, select the library that needs to be changed.
+1. If necessary, change _Index Family_. If selected, the indices will be erase.
+1. Select the correct index under _Indicies_. If the family supports it, another drop down will appear.
```

Comment:
"If the family supports it" sounds very mob-like. "If the index family supports dual-barcoding"?